src/logger_dispatch.jl

```diff
@@ -38,6 +38,22 @@ See also: [`Base.propertynames`](@ref)
 """
 logable_propertynames(val::Any) = propertynames(val)
 
+## Default unpacking of key-value dictionaries
+function preprocess(name, dict::Dict, data)
+    i=1
+    for k=keys(dict)
```

Comment:
Can use `enumerate` for this.
Then can delete the `i=1` and `i += 1`
```suggestion
    for (i,k) in enumerate(keys(dict))
```